.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -44,10 +65,15 @@ run_sanity_check() {
         cd release/util
         python sanity_check.py
     )
+    (
+        cd release/util
```

Comment:
remove these 3 lines? we do not need to `cd release/util` twice.

Reply:
fixed!